kyo-combinators/shared/src/main/scala/kyo/Combinators.scala

```diff
@@ -386,6 +376,24 @@ extension [A, S, E](effect: A < (Abort[E] & S))
     ): A < (S & reduce.SReduced) =
         Abort.run[Throwable](effect.asInstanceOf[A < (Abort[Throwable | ER] & S)])
             .map(_.fold(e => throw e.getFailure)(identity))
+
+    /** Catches any Aborts and panics instead
+      *
+      * @return
+      *   A computation that panics instead of catching Abort effect failures
+      */
+    def orDie(
+        using
+        ct: SafeClassTag[E],
+        fl: Flat[A],
+        frame: Frame
+    ): A < S =
+        Abort.run[E](effect).map(
+            _.fold(_.getFailure match
+                case thr: Throwable => throw thr
+                case other          => throw PanicException(other)
+            )(identity)
+        )
```

Comment:
I don't think we should throw, we should `Abort.panic`. Long term it think core effect handling will need to be aware of Panics...